### ISSUE TITLE
These commands run on which host?

### ISSUE THREAD

**Issue by ghost** [sekka1/mesosphere-docker#8]
Multi-node Setup
###### 

4.Start Marathon
6.Goto the Meso's Web page
7.Goto the Marathon's Web page
###### 

These commands run on which host?

**Comment by sekka1** [comment]
I dont think this works anymore and I have not used mesosphere in over a year.
